presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -164,6 +168,15 @@ public void testCharTrailingSpace()
 
         assertEquals(getQueryRunner().execute("SELECT * FROM char_trailing_space WHERE x = char ' test'").getRowCount(), 0);
 
+        QueryRunner queryRunner = createQueryRunnerWithLegacyCoercion();
```

Comment:
wrap with try-with-resources  to close the runner

(see another comment for more info)